### ISSUE TITLE
Testing real burst.

### ISSUE THREAD

**Issue by xyIsHere** [NaamaPearl/nan#2]
Dear author,

Could you provide a detailed instruction on testing a real burst?

Thanks.

**Comment by NaamaPearl** [comment]
I don't have an organized script for this, but I will detail my steps:

*Extracting poses: 
I first processed the DNGs into sRGB images with rawpy (those are used only for the poses calculation). For scenes with high levels of noise COLMAP fails to calculate accurate poses, so we used Metashape. When I used COLMAP, I used the scripts in https://github.com/Fyusion/LLFF/blob/master/imgs2poses.py  to process the poses. I rearranged the images in the same way as the LLFF dataset and used the same dataloaders for handling the data in my code.

*Undistortion: 
When the undistort parameters of the pose's calculation were accurate, I used opencv.Undistort. These were most of the time not accurate, and it was better to leave the images distorted. In some cases, I found out that the network is able to slightly undistort the image in the generation process.

*Reading raw for processing: 
I used rawpy without any color correction and read the raw data in uint16. I didn't save and reopen to avoid numerical errors. I extracted the noise parameters from the EXIF and adjusted it based on the brightness adjustment that I applied to the images. This part is merely an estimation. I found out that the exact values of them didn't change my results much but were highly affected the other algorithms I ran.
